single_cell/workflows/cohort_qc/tasks.py

```diff
@@ -45,52 +44,91 @@ def generate_segmental_copynumber(hmmcopy_files, segmental_cn, sample):
     )
 
 
-def merge_mafs(germline_mafs, somatic_mafs, merged_maf):
+def format_maf_types(maf):
+    """Format maf.
+
+    Args:
+        maf ([pandas.DataFraame]): [maf data]
+
+    Returns:
+        [pandas.DataFrame]: [formatted maf]
+    """
+    float_cols = [
```

Comment:
could you name this var and floatcols something that's more obvious and less confusing.